lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -446,6 +466,33 @@ instance FromHttpApiData Iso8601Time where
 instance ToHttpApiData Iso8601Time where
     toUrlPiece = toText
 
+instance FromText ApiEpochNumber where
+    fromText txt = case txt of
+        "latest" -> Right (ApiEpochNumber "latest")
```

Comment:
🤔 What about defining `ApiEpochNumber` as something like:

```haskell
data ApiEpochNumber = ApiEpochNumberLatest | ApiEpochNumber EpochNumber
```

Reply:
It seems like when actually using `ApiEpochNumber`, you'd have to re-parse the `Text` into either the notion of latest or an `EpochNumber` anyway.

Reply:
I was persuaded and used the propose type. It is more precise and better reflect situation